### PR TITLE
implement VGG16 with example

### PR DESCRIPTION
**Pull request by drbh**
This PR adds the VGG16 model to `candle-transformers` and a small example to the `candle-example` folder.

Example output:
vgg16test

Please let me know if I should make any changes! Thanks for this awesome repo!!

### CODE REVIEW

File: candle-examples/examples/vgg16/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+```bash
+python3 export_models.py
```
Comment (LaurentMazare) [inline candle-examples/examples/vgg16/README.md:4]: We would rather get all the examples to be self-contained through the use of the `hf-hub` crate, see the other examples. In this case, you can find some sample model weights [here](https://huggingface.co/timm/vgg16.tv_in1k/tree/main).

File: candle-examples/examples/vgg16/assets/meerkat.jpg
Comment (LaurentMazare) [inline candle-examples/examples/vgg16/assets/meerkat.jpg:1]: Let's avoid adding new images to the repo if possible, it's already on the large side. Maybe you can just use one of the existing image for the example readme.

**Review by LaurentMazare — COMMENTED** [review]
Thanks for the PR. Rather than just adding vgg16, it would be nicer to add vgg13/16/19 in one go (and so rename the files from e.g. `vgg16.rs` to `vgg.rs`).

**Comment by drbh** [comment]
> Thanks for the PR. Rather than just adding vgg16, it would be nicer to add vgg13/16/19 in one go (and so rename the files from e.g. `vgg16.rs` to `vgg.rs`).

thanks for the quick feedback! 🙏 I agree it makes sense to cover those models as well; will push changes shortly

**Pull request closed without merging**